packages/angular_devkit/build_angular/test/hello-world-app/tsconfig.json

```diff
@@ -8,12 +8,10 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "skipLibCheck": true,
     "target": "es2022",
     "module": "es2022",
     "useDefineForClassFields": false,
-    "typeRoots": [
```

Comment:
In TS 5.1 the presence of `typeRoots` means that only those specific directories will be checked. Previously TS was also going up the file tree. I had to remove this, because the type root doesn't contain types for `@anguar/localize`.